### ISSUE TITLE
"import numpy" isnt working with "pip install numpy" in anaconda enviroment (py 3.8.5)

### ISSUE THREAD

**Issue by Gurren-Joestar** [numpy/numpy#17840]
Hi, im having some issues installing numpy trought pip in a anaconda enviroment (python 3.8.5), my numpy version is 1.19.4
```
(py385) C:\Windows\system32>pip list
Package         Version
--------------- -------------------
certifi         2020.6.20
cycler          0.10.0
kiwisolver      1.3.1
matplotlib      3.3.3
numpy           1.19.4
Pillow          8.0.1
pip             20.2.4
pyparsing       2.4.7
python-dateutil 2.8.1
setuptools      50.3.1.post20201107
six             1.15.0
wheel           0.35.1
wincertstore    0.2
```

### Steps to reproduce:

Install anaconda 3 with python 3.8.3

Make a new enviroment (anaconda prompt) :
`conda create -n py385 py=3.8.5`
The enviroment should be empty after doing that

Activate the new conda env
`conda activate py385`

Install numpy through pip (py385)
`pip install numpy`

Importing numpy in python
`>>>import numpy as np`

### Error message:

 ```
** On entry to DGEBAL parameter number  3 had an illegal value
 ** On entry to DGEHRD  parameter number  2 had an illegal value
 ** On entry to DORGHR DORGQR parameter number  2 had an illegal value
 ** On entry to DHSEQR parameter number  4 had an illegal value
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\<name>\.conda\envs\py385\lib\site-packages\numpy\__init__.py", line 305, in <module>
    _win_os_check()
  File "C:\Users\<name>\.conda\envs\py385\lib\site-packages\numpy\__init__.py", line 302, in _win_os_check
    raise RuntimeError(msg.format(__file__)) from None
RuntimeError: The current Numpy installation ('C:\\Users\\<name>\\.conda\\envs\\py385\\lib\\site-packages\\numpy\\__init__.py') fails to pass a sanity check due to a bug in the windows runtime. See this issue for more information: https://tinyurl.com/y3dm3h86

```

**Comment by mattip** [comment]
> See this issue for more information: https://tinyurl.com/y3dm3h86

Did you visit the URL? 
Did you have any questions about the fix to use `conda install numpy==1.19.3` instead of `conda install numpy`?

**Comment by Gurren-Joestar** [comment]
> > See this issue for more information: https://tinyurl.com/y3dm3h86
> 
> Did you visit the URL?
> Did you have any questions about the fix to use `conda install numpy==1.19.3` instead of `conda install numpy`?

Installing numpy==1.19.3 solve the problem
